src/app/views/apps-management/app-management/app-management.controller.coffee

```diff
@@ -115,6 +115,8 @@ angular.module 'mnoEnterpriseAngular'
 
         # ********************** Initialize *********************************
         vm.init = ->
+          return $state.go('home.impac') unless MnoeConfig.isProvisioningEnabled()
```

Comment:
Should this also chek if AppManagement is enabled?

Reply:
And rather than doing it this way, it should be done in the router (index.route.coffee) so that we use the `$urlRouterProvider.otherwise` block rather than manually changing state